CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.1.4]
+
+### Fixed
+
+- Fixed a bug on ARM64 hosts where the upper 64bits of the V0-V31 FL/SIMD
```

Comment:
Nit space:
```suggestion
- Fixed a bug on ARM64 hosts where the upper 64 bits of the V0-V31 FL/SIMD
```

Reply:
I copied it from main. It may be worth fixing it there instead.